protobuf/lib/src/protobuf/field_info.dart

```diff
@@ -221,7 +221,12 @@ String _unCamelCase(String name) {
 }
 
 class MapFieldInfo<K, V> extends FieldInfo<PbMap<K, V>?> {
+  /// Key type of the map. Per proto2 and proto3 specs, this needs to be an
```

Comment:
Perhaps write that it is interpreted as the FieldInfo.type field (if that is indeed the case).

Reply:
Done.